### PR TITLE
Enabled the CLI Behave tests on Ubuntu18.04

### PR DESCRIPTION
**Pull request by nkalampalikis**
We now run the behave CLI tests on Ubuntu18.04.  We skip tests that were falling when run on that platform.  We plan to fix these subsequently.

There are a few other limitations of this PR:
- we do not run the `check_centos` job on Ubuntu18.04
- it is not possible to run just the CLI tests on Ubuntu18.04 with this PR. We deferred this, as doing seemed to drag this PR on too long.

After responding to PR comments, we will also commit the generated yml files as well. 

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@nkalampalikis Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=7957)!

[Click here](https://cla.pivotal.io/sync/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=7957) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@nkalampalikis Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=7957)!

[Click here](https://cla.pivotal.io/sync/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=7957) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@nkalampalikis Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=7957)!

File: gpMgmt/test/behave/mgmt_utils/gpconfig.feature [hunk 1]
```diff
@@ -80,10 +80,14 @@ Feature: gpconfig integration tests
...
         | escaped single quote                   | application_name             | string     | boo        | "\'"      | '\\'''     | \'         | "\'"              | '\\'''                 | "\'"         | '\\'''            | \'                |
         | multiple quoted single quotes          | application_name             | string     | boo        | "''''"    | '''''''''' | ''''       | "''"              | ''''''                 | "'"          | ''''              | '                 |
-        | utf-8 works                            | search_path                  | string     | boo        | Ομήρου    | 'Ομήρου'   | Ομήρου     | Ομήρου            | 'Ομήρου'               | Ομήρου       | 'Ομήρου'          | Ομήρου            |
 #       | integer with time unit with spaces     | statement_timeout            | int w/unit | 2min       | "'7 min'" | '7 min'    | 7min       | "'7 min'"         | '7 min'                | "'7 min'"    | '7 min'           | 7min              |
 # 'Integer with time unit with spaces' fails because the live server parses '7 min' as 7min, and our comparison logic does not handle this correctly.
 
+    @skip_fixme_ubuntu18.04
+    Examples:
```
Comment (jchampio) [inline gpMgmt/test/behave/mgmt_utils/gpconfig.feature]: You can tag `Examples`? That's so cool.

Comment (dkrieger-pvtl) [reply gpMgmt/test/behave/mgmt_utils/gpconfig.feature]: Yes...the intended use case is for different development stages: dev and test, for instance.  But you can use it for this too.

**Comment by dkrieger-pvtl** [comment]
> * I've been incorrectly tagged as a co-author on the second commit.
> * A body for the second commit, explaining the failures that are being skipped, would be good.
> * You'll want to coordinate with #7911; they're going to conflict with the changes here.

Yes, we'll correct the first two items before we commit.  We'll let #7911 go in first as our merge will be trivial for them and it's also quite easy for us to re-test.

**Pull request merged**